access-control-service-sql-impl/src/main/java/org/eclipse/tractusx/semantics/accesscontrol/sql/model/converter/CustomAccessRuleMapper.java

```diff
@@ -19,16 +19,28 @@
  ******************************************************************************/
 package org.eclipse.tractusx.semantics.accesscontrol.sql.model.converter;
 
+import org.apache.commons.lang3.StringUtils;
 import org.eclipse.tractusx.semantics.accesscontrol.sql.model.AccessRule;
 import org.mapstruct.AfterMapping;
 import org.mapstruct.MappingTarget;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Component;
 
 @Component
 public class CustomAccessRuleMapper {
 
+   private final String ownerTenant;
+
+   public CustomAccessRuleMapper( @Value( "${registry.idm.owning-tenant-id:}" ) String ownerTenant ) {
+      this.ownerTenant = StringUtils.stripToNull( ownerTenant );
+   }
+
    @AfterMapping
-   public void calledWithSourceAndTarget(Object anySource, @MappingTarget AccessRule target) {
+   public void calledWithSourceAndTarget( Object anySource, @MappingTarget AccessRule target ) {
```

Comment:
anySource is not used and can be removed or?

Reply:
Thank you, fixed it!